aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -0,0 +1,482 @@
+# Marketing App Resources
+# This template is manually deployed to create/update the Marketing App's AWS resources.
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Marketing App AWS resources
+
+Parameters:
+  Environment:
+    Type: String
+    Default: "test"
+    Description: "Environment being deployed to (production or test)"
+  ContainerImageHashDigest:
+    Type: String
+    Description: "The sha256sum of the marketing container image."
+
+Resources:
+  # Route 53 root hosted zone for marketing.dev-code.org
+  RootHostedZone:
+    Type: AWS::Route53::HostedZone
+    Properties:
+      Name: cfn-marketing.dev-code.org.
+
+  # TLS Cert for the ECS ALB
+  TLSCertificate:
```

Comment:
This is perhaps a nitpick on the structure of this document, but I think the most readable way to structure a clouformation template is to go "outside-in" and to push all supplementary resources down. This might look like:

- DNS
- Cloudfront Distro
- ELB
- ECS Service
- ECS Task Definition


Everything else helps further refine those, and those are the ones that are most likely to be edited (or to be the start of an investigation.